book/08-customizing-git/sections/attributes.asc

```diff
@@ -329,9 +329,9 @@ The substitutions can include for example the commit message and any `git notes`
 [source,console]
 ----
 $ echo '$Format:Last commit: %h by %aN at %cd%n%+w(76,6,9)%B$' > LAST_COMMIT
-$ git commit -am 'export-subst uses git log's custom formatter
+$ git commit -am 'export-subst uses git log'\''s custom formatter
```

Comment:
Hum, I have another guess:

    $ git commit -am 'export-subst uses git log\'s custom formatter'

with a quote at the end.